apps/wing/src/commands/consoleAppPath.ts

```diff
@@ -0,0 +1,4 @@
+export async function consoleAppPath() {
+  const path = require.resolve("@wingconsole/app");
+  console.log(path);
+}
```

Comment:
Since this is hidden, maybe add a comment on what this is used for

Reply:
removed for now 👍